sql/professor_isf_edita_repositorio.sql

```diff
@@ -0,0 +1,23 @@
+-- Tabela que refere-se as informações para que seja possível o Professor ISF editar os materiais de um repositório
+
+CREATE TABLE professor_isf_edita_repositorio (
+	id SERIAL PRIMARY KEY,
+	professor_isf CHAR(11) NOT NULL,
+	cursista CHAR(11) NOT NULL,
+	codigo_repositorio INTEGER NOT NULL,
+	
+	CONSTRAINT fk_professor_isf 
+		FOREIGN KEY (professor_isf) 
+			REFERENCES professor_isf (cpf),
+	
+	CONSTRAINT fk_cursista 
+		FOREIGN KEY (cursista) 
```

Comment:
A tabela cursista não existe. Por favor, adicionem ela e solicitem a review novamente. Se cursista não for do seu grupo, crie uma tabela apenas com o identificador.